samples/testing-frameworks/appium/client-side/ruby/bitbar_appiumdriver_ios.rb

```diff
@@ -33,17 +33,17 @@ def log(msg)
 ##  Set other parameters if needed, see more in README
 ## 
 desired_capabilities_cloud = {
-    'device' => 'iphone',
-    'bitbar_apiKey' => bitbar_api_key,
-    'bitbar_project' => 'Appium iOS Project1',
-    'bitbar_description' => 'Appium project description',
-    'bitbar_testrun' => 'Test Run 1',
-    'bitbar_device' => bitbar_device,
-    'bitbar_target' => 'ios',
-    'deviceName' => 'iPhone device',
     'platformName' => 'iOS',
-    'bundleId' => 'com.bitbar.testdroid.BitbarIOSSample',
-    'automationName' => 'XCUITest'
+    'appium:automationName' => 'XCUITest',
+    'appium:bundleId' => 'com.bitbar.testdroid.BitbarIOSSample',
```

Comment:
For some reason wrapping it into `appium:options` didn't work in iOS case. Same case in `bitbar_webdriver_ios.rb`

Reply:
This was an issue in Appium2 beta - https://github.com/appium/appium/issues/18191
It was fix in https://github.com/appium/appium/releases/tag/appium%402.0.0-beta.56

We can switch to `appium:options` after upgrading Appium in cloud